mods/ra/sequences/structures.yaml

```diff
@@ -131,12 +131,16 @@ powr:
 
 apwr:
 	idle:
+		Offset: 0,-10
```

Comment:
Yeah, this change is basically similar to the airfield and barracks offset changes, it's supposed to make the visuals match the occupied cells a little better.